core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/resourcebuffer/ResourceBufferSpec.scala

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.dsde.rawls.dataaccess.resourcebuffer
+
+import akka.http.scaladsl.model.headers.OAuth2BearerToken
+import com.typesafe.config.ConfigFactory
+import org.broadinstitute.dsde.rawls.config.ResourceBufferConfig
+import org.broadinstitute.dsde.rawls.mock.MockResourceBufferDAO
+import org.broadinstitute.dsde.rawls.model.{GoogleProjectId, ProjectPoolType, RawlsUserEmail, RawlsUserSubjectId, UserInfo}
+import org.broadinstitute.dsde.rawls.resourcebuffer.ResourceBufferService
+import org.scalatest.flatspec.AsyncFlatSpec
+
+class ResourceBufferSpec extends AsyncFlatSpec {
+
+
+  val userInfo = UserInfo(RawlsUserEmail("owner-access"), OAuth2BearerToken("token"), 123, RawlsUserSubjectId("123456789876543212345"))
```

Comment:
I am assuming the `RawlsUserSubjectId` is not important here? Could we have it generate a random number string instead? (Just to show that it is irrelevant?)